### PR TITLE
new alert for permissions

### LINKED ISSUE #123: [Alert] New Alert Type for Permissions
#### Mockups

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

### PR DESCRIPTION
**Pull request by iArchitSharma**
**Notes for Reviewers**
New alert type added
**USE** 
Use the following Syntax to add a Alert:
`{{< alert type="success" title="Note" >}} Your Note {{< /alert >}}`

  - `type="danger"`: Alert used to indicate something related to security.
  - `type="info"`: Alert used to write some information.
  - `type="warning"`: Alert used to indicate a warning that might need attention.
  - `type="neutral"`: Alert used to indicate a neutral information.
  - `type="success"`: Alert used to indicate a positive information.

By default, if you don't specify a title `title="Your Title"`, the title will be the same as the type name.
image

This PR fixes #123 

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@iArchitSharma will you add this new style and its intended use to a Layer5 Docs Contributing guide?

**Review by leecalcote — APPROVED** [review]

**Comment by iArchitSharma** [comment]
sure @leecalcote

**Comment by leecalcote** [comment]
_Darn you, DNS_

**Comment by iArchitSharma** [comment]
Done @leecalcote

File: README.md [hunk 1]
```diff
@@ -54,6 +54,14 @@ To display images in a pop-up modal, use the following syntax: `![alt text](/pat
 
 **Note-> For images to show correctly on subpages, always provide the complete image path**
 
+### Adding Alert to the site
+Use the following Syntax to add a Alert:
```
Comment (leecalcote) [inline README.md:58]: @iArchitSharma make sure that this is a general description of the manner in which our docs support info / notes / tips / alerts and so on, not just a permission alert.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (leecalcote) [reply README.md:58]: Make a complete list of supported styles with example use, please.

Comment (leecalcote) [reply README.md:58]: This same thing already be done and measure the docs. If you don't find it, please create one. But, there should be one already.

**Pull request merged**